pyemma/msm/api.py

```diff
@@ -555,7 +560,8 @@ def estimate_markov_model(dtrajs, lag, reversible=True, statdist=None,
             import warnings
             warnings.warn("Values for statdist, maxiter or maxerr are ignored if OOM-correction is used.")
         oom_msm = _OOM_MSM(lag=lag, reversible=reversible, count_mode=count_mode,
-                           sparse=sparse, connectivity=connectivity, dt_traj=dt_traj)
+                           sparse=sparse, connectivity=connectivity, dt_traj=dt_traj,
+                           trajectory_bootstrap=trajectory_bootstrap)
```

Comment:
How about this:
``rank_C0 = 'bootstrap_counts'``
where ``bootstrap_counts`` and ``bootstrap_trajs`` are options. This allows us to choose other rank selection methods later. I'd choose bootstrap_counts as default as it always works, while bootstrap_trajs is only meaningful if you have enough trajectories.